src/Queries/Dates/February.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Here's a suggestion: I suspect PSR4 doesn't allow us to put these classes into a common file, correct? However they don't really need to be classes. How about we add a single `ConstantQuery` class. The constructor of a `ConstantQuery` takes a `TermTermType` value, and its `getTermType()` method is implemented so that it returns the `TermTermType` that was specified in the constructor.

Then we can remove all these classes for the days of the week and the months, and instead just have the global `r\monday()`, ... and `r\january()` methods return an instance of `ConstantQuery` constructed with the correct term type.